examples/api/houseprice.py

```diff
@@ -10,6 +11,11 @@
 lr.fit(df[["BedroomAbvGr", "YearBuilt"]], df["SalePrice"])
 
 
+@app.get("/")
+async def docs_redirect():
+    return Response("Opening the docs UI", status_code=307, headers={"location": "/docs"})
```

Comment:
Since we are explicitly setting the status let's use something more typical, I suggest 302

Reply:
okie